cpp/src/io/comp/debrotli.cu

```diff
@@ -1966,7 +1972,16 @@ static __device__ void ProcessCommands(debrotli_state_s *s, const brotli_diction
 }
 
 
-
+/**
+ * @brief Brotli decoding kernel
+ * See https://tools.ietf.org/html/rfc7932
+ *
+ * @param inputs[in] Source/Destination buffer information per block
+ * @param outputs[out] Decompressor status per block
+ * @param scratch Intermediate device memory heap space (will be dynamically shared between blocks)
+ * @param scratch_size Size of scratch heap space (smaller sizes may result in serialization between blocks)
+ * @param count Number of blocks to decompress
+ **/
 // blockDim {128,1,1}
```

Comment:
Will this comment cause the above doc to be detached from the below kernel? Maybe this comment can be absorbed into the doc.

Reply:
Generally, the columnar io kernels have a simple, well documented C-like interface in the headers (that's presumably where the documentation will be generated from). I think this issue is more for internal documentation in the code itself, as many of these kernels are relatively complex (particularly compression/decompression/encode/decode).
Will absorb the various blockDim comments in the description block.

Reply:
To clarify: none of these kernels are directly exported in the cuh/hpp headers: instead there is a C-style host API (at the end of the file) that just launches the corresponding kernel(s). I don't think that would cause a conflict in the documentation, though it may cause kernels to be documented where they previously were not (not sure if that would cause an over-documentation problem ?)

Reply:
Hmm. Looking at the auto-generated libcudf docs, it seems like the main doc is from python APIs, and the libcudf doc appears to come from things exposed from a subset of cpp/include (I couldn't find anything related to low-level io kernels)